src/installing-workflow/configuring-object-storage.md

```diff
@@ -25,172 +25,26 @@ If you provide credentials with sufficient access to the underlying storage, Wor
 
-### Step 4: Configure Workflow charts
+### Step 4: Configure Workflow Chart
 
-Operators should configure object storage by either populating a set of environment variables or editing the the Helm Classic parameters file before running `helmc generate`. Both options are documented below:
+Operators should configure object storage by editing the Helm values file before running `helm install`. To do so:
 
-**Option 1:** Using environment variables
-
-After setting a `STORAGE_TYPE` environment variable to the desired object storage type ("s3", "gcs", "azure", or "swift"), set the additional variables as required by the selected object storage:
-
-| Storage Type | Required Variables                                                                                                                                          | Notes                                                                                               |
-| ---          | ---                                                                                                                                                         | ---                                                                                                 |
-| s3           | `AWS_ACCESS_KEY`, `AWS_SECRET_KEY`, `AWS_REGISTRY_BUCKET`, `AWS_DATABASE_BUCKET`, `AWS_BUILDER_BUCKET`, `S3_REGION`                                         | To use [IAM credentials][aws-iam], it is not necessary to set `AWS_ACCESS_KEY` or `AWS_SECRET_KEY`. |
-| gcs          | `GCS_KEY_JSON`, `GCS_REGISTRY_BUCKET`, `GCS_DATABASE_BUCKET`, `GCS_BUILDER_BUCKET`                                                                          |                                                                                                     |
-| azure        | `AZURE_ACCOUNT_NAME`, `AZURE_ACCOUNT_KEY`, `AZURE_REGISTRY_CONTAINER`, `AZURE_DATABASE_CONTAINER`, `AZURE_BUILDER_CONTAINER`                                |                                                                                                     |
-| swift        | `SWIFT_USERNAME`, `SWIFT_PASSWORD`, `SWIFT_AUTHURL`, `SWIFT_AUTHVERSION`, `SWIFT_REGISTRY_CONTAINER`, `SWIFT_DATABASE_CONTAINER`, `SWIFT_BUILDER_CONTAINER` | To specify tenant set `SWIFT_TENANT` if the auth version is 2 or later.                             |
-
-!!! note
-	These environment variables should be set **before** running `helmc generate` in Step 5.
-
-**Option 2:** Using template file `tpl/generate_params.toml` available at `$(helmc home)/workspace/charts/workflow-v2.8.0`
-
-* Edit Helm Classic chart by running `helmc edit workflow-v2.8.0` and look for the template file `tpl/generate_params.toml` (make sure you have the `$EDITOR` environment variable set with your favorite text editor)
-* Update the `storage` parameter to reference the platform you are using, e.g. `s3`, `azure`, `gcs`, or `swift`
+* Fetch the Helm values by running `helm inspect values deis/workflow > values.yaml`
```

Comment:
@bacongobbler I'm having trouble with this approach:
```
$ helm inspect values workflow/workflow > values.yaml

$ helm install workflow/workflow -f values.yaml --namespace=deis
Fetched workflow/workflow to workflow-v2.8.0.tgz
Error: error converting YAML to JSON: yaml: line 10: did not find expected <document start>

$ helm version
Client: &version.Version{SemVer:"v2.0.0", GitCommit:"e77d564b880c4436049f0723083d67ef810729b2", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.0.0", GitCommit:"51bdad42756dfaf3234f53ef3d3cb6bcd94144c2", GitTreeState:"clean"}
```
The reason seems to be that the `helm inspect` outputs the following line first: `Fetched workflow/workflow to workflow-v2.8.0.tgz` which is then the first line in `values.yaml`.  If this approach is pursued, that first line needs to be removed...